src/apps/btc/btc_sign.c

```diff
@@ -306,7 +305,15 @@ app_btc_sign_error_t app_btc_sign_input(
         return _error(APP_BTC_SIGN_ERR_STATE);
     }
     if (request->sequence != 0xffffffff) {
-        return _error(APP_BTC_SIGN_ERR_INVALID_INPUT);
+        // rbf without a locktime is ill-defined
+        if (request->sequence == 0xffffffff - 2 && _locktime != 0) {
+            _rbf = 1;
+        }
+        // sequence bytes indicate a locktime, so check that it is defined
+        else if (request->sequence == 0xffffffff - 1 && _locktime != 0) {
```

Comment:
I left out the coin check, do you think we really need it? Electrum-ltc still sets the sequence bytes and locktime for example.

Reply:
yeah we need it. you can't ask the user to confirm that they are using rbf in litecoin. 

i would even add a `_handle_rbf` bool that is true only for Bitcoin. reason:

we should think about even confirming with the user when they are *not* using rbf (at least in the expert mode i proposed in my blog post), and that should not slip into Litecoin either, so separating rbf tracking from whether it should be confirmed in the first place seems more robust. 

instead of the `_handle_rbf` var, maybe it even makes sense to `#if` guard all the rbf stuff using the `APP_BTC` macro.

Reply:
This seems a bit premature if we don't know yet how the expert mode will look like. How about just adding an `#if defined(APP_BTC)` guard around the `_rbf=true`?

Reply:
Added `#if defined(APP_BTC)` guard around the `_rbf=true`